static/styles.css

```diff
@@ -278,8 +284,34 @@ input[type="checkbox"] {
 .results-list__image {
   width: 140px;
   height: 120px;
-  padding: 0 var(--space-medium);
-  padding-left: 0;
+  /* padding: 0 var(--space-medium);
```

Comment:
@bridgetburke padding vs. margins? I switched to margins because _sometimes_ the images aren't 140px wide and styling the blanks got weird. The blank div is there for records that don't have an image thumbnail: https://peaceful-goldberg-4bd4d0.netlify.app/samples/qbat/m/mlibrary1ic/q_mlibrary1ic_x_mlibrary1ic_publisher___tucson_ariz__nazraeli_press___0001/